<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>585</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>585</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Youtube Downloader 8v</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>YtExtraFiles/YtDownloader.png</normaloff>YtExtraFiles/YtDownloader.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(200, 200, 200);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>586</width>
     <height>405</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>560</width>
     <height>405</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #181818;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="mainWidget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #181818;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="search_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #181818;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="search_input">
            <property name="toolTip">
             <string notr="true">Coloque um link de um video ou pesquise videos do youtube</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolTip {
color: white;
}
 * {
background-color: #202020;
color: rgb(200, 200, 200);
border: 1px solid #272727;
}
*:hover {
border: 1px solid #353535;
}
*:focus {
border: 1px solid #404040;
}

</string>
            </property>
            <property name="placeholderText">
             <string notr="true">URL OU PESQUISAR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="search_spinbox">
            <property name="toolTip">
             <string notr="true">Quantidade de videos a serem pesquisados
</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolTip {
color: white;
}
 * {
background-color: #202020;
color: rgb(255, 255, 255);
border: 1px solid #272727;
}
*:hover {
border: 1px solid #353535;
}
*:focus {
border: 1px solid #404040;
}</string>
            </property>
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>40</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>18</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="search_btn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolTip {
color: white;
}
 * {
background-color: #202020;
color: rgb(200, 200, 200);
}
</string>
            </property>
            <property name="text">
             <string notr="true">Procurar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="download_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #181818;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="searchBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolTip {
color: white;
}
 * {
background-color: #202020;
color: rgb(200, 200, 200);
border: 1px solid #272727;
}
*:hover {
border: 1px solid #353535;
}
*:focus {
border: 1px solid #404040;
}
QComboBox QAbstractItemView {
outline: none;
}
</string>
            </property>
            <property name="duplicatesEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="downloadBtn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolTip {
color: white;
}
 * {
background-color: #202020;
color: rgb(200, 200, 200);
}
</string>
            </property>
            <property name="text">
             <string notr="true">Baixar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="download_table">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>220</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
* {
background-color: #202020;
color: rgb(200, 200, 200);
border: 1px solid #272727;
}

QHeaderView::section{
background-color: #202020;
color: rgb(200, 200, 200);

}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>0</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>112</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">Título</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">Tamanho</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">Velocidade</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">Baixado</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">Status</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="config_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>57</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #181818;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="qualities_box">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string notr="true">Opções de download</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolTip {
color: white;
}
 * {
background-color: #202020;
color: rgb(200, 200, 200);
border: 1px solid #272727;
}
*:hover {
border: 1px solid #353535;
}
*:focus {
border: 1px solid #404040;
}

QComboBox QAbstractItemView {
outline: none;
}
</string>
            </property>
            <property name="currentText">
             <string notr="true">Apenas o áudio</string>
            </property>
            <property name="currentIndex">
             <number>6</number>
            </property>
            <item>
             <property name="text">
              <string notr="true">Melhor qualidade possível</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">1080p</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">720p</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">480p</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">360p</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Pior qualidade possível</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Apenas o áudio</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>63</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="folder_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string notr="true">Selecione a pasta que o arquivo será baixado</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolTip {
color: black;
}
 * {
background-color: rgb(103, 103, 103);
color: rgb(255, 255, 255);
}

</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>YtExtraFiles/folder.ico</normaloff>YtExtraFiles/folder.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="folders_box">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>240</width>
              <height>20</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string notr="true">Locais para salvar o arquivo</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolTip {
color: white;
}
 * {
background-color: #202020;
color: rgb(200, 200, 200);
border: 1px solid #272727;
}
*:hover {
border: 1px solid #353535;
}
*:focus {
border: 1px solid #404040;
}
QComboBox QAbstractItemView {
outline: none;
}</string>
            </property>
            <property name="currentText">
             <string notr="true"/>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
background-color: #202020;
color: rgb(200, 200, 200);
border: 1px solid #272727;
}

*:hover {
background-color: #303030;
color: rgb(220, 220, 220);
}</string>
   </property>
   <widget class="QMenu" name="menuConfig">
    <property name="toolTip">
     <string notr="true">Ficar na frente</string>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
background-color: #202020;
color: rgb(200, 200, 200);
}
*:hover {
background-color: #303030;
color: rgb(220, 220, 220);
}
*:focus {
background-color: #303030;
color: rgb(220, 220, 220);}</string>
    </property>
    <property name="title">
     <string notr="true">Configurações</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="toggle_window_btn"/>
    <addaction name="reset_folders_btn"/>
    <addaction name="reset_suggestions_btn"/>
   </widget>
   <addaction name="menuConfig"/>
  </widget>
  <action name="toggle_window_btn">
   <property name="text">
    <string notr="true">Ficar na frente</string>
   </property>
   <property name="iconText">
    <string notr="true">Ficar na frente</string>
   </property>
   <property name="toolTip">
    <string notr="true">Ficar na frente</string>
   </property>
  </action>
  <action name="reset_folders_btn">
   <property name="text">
    <string notr="true">Resetar pastas</string>
   </property>
   <property name="iconText">
    <string notr="true">Resetar pastas</string>
   </property>
   <property name="toolTip">
    <string notr="true">Resetar pastas</string>
   </property>
  </action>
  <action name="reset_suggestions_btn">
   <property name="text">
    <string>Resetar sugestões</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>search_input</sender>
   <signal>returnPressed()</signal>
   <receiver>search_btn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
